gui/modes/CommandPressureMode.qml

```diff
@@ -0,0 +1,308 @@
+import QtQuick 2.11
+import QtQuick.Layouts 1.3
+import QtQuick.Controls 2.4
+
+import "../controls"
+import ".."
+
+Mode {
+    Timer // TODO: Make data sources be updated as we get data
+    {
+        id: refreshTimer
+        interval: 1 / 15 * 1000 // 60 Hz
```

Comment:
I know this was the same in the original code - but this is actually 15Hz, not 60Hz, right?